ci/docker/android-emulator/hardware/config_29.ini

```diff
@@ -23,7 +23,7 @@ hw.sensors.pressure=no
 hw.sensors.humidity=no
 hw.sensors.magnetic_field_uncalibrated=no
 hw.sensors.gyroscope_uncalibrated=no
-image.sysdir.1=system-images/android-29/google_apis/x86_64/
```

Comment:
Why do we switch to x86? Is it to check crashes in qemu?
It'd be good also to compare performance.

Reply:
i wanted to try x86, because of Anton's article 
https://malinskiy.medium.com/android-ci-with-kubernetes-684713a83eec

```
The emulator we’re using is the x86 version, not the x86_64: after benchmarking with AnTuTu we’ve found x86 version to be noticeably faster.
```

Reply:
reverted this change, shouldn't be mixed with other updates imo

and according to @materkey 's benchmarks it's ~even now, _64 slightly better in fact